tests/stamp_test.py

```diff
@@ -36,7 +36,7 @@ class StampTest(unittest.TestCase):
   # CI run and have the test run a few seconds later. Ideally, this would
   # be no greater than the expected total CI run time, but the extra margin
   # does not hurt.
-  ALLOWED_DELTA_FROM_NOW = 10000  # seconds
+  ALLOWED_DELTA_FROM_NOW = 30000  # seconds
```

Comment:
It might be nice to make this a day (86400s) to make it a little more concrete.

Reply:
Does that make it too wide? I'm really at a loss. 
No matter how wide I make it, I find it still fails when I run it inside Google. Our caching previously built things is good enough to keep it for days. :-(